dotnet/src/SemanticKernel.Abstractions/AI/ChatCompletion/ChatCompletionServiceChatClient.cs

```diff
@@ -195,7 +195,7 @@ public void Dispose()
 
         if (options.Tools is { Count: > 0 })
         {
-            var functions = options.Tools.OfType<AIFunction>().Select(f => new AIFunctionKernelFunction(f));
+            var functions = options.Tools.OfType<AIFunction>().Select(KernelFunction.FromAIFunction);
```

Comment:
To keep consistency with existing `AsAiFunction`.

```suggestion
            var functions = options.Tools.OfType<AIFunction>().Select(aiFunction => aiFunction.AsKernelFunction());
```